beacon/validation/request.py

```diff
@@ -136,6 +136,8 @@ async def clean_fields(self, req, qparams):
 
             if key == 'targetIdReq':
                 qval = req.match_info.get('target_id_req')
+            if key == 'cohortId':
```

Comment:
Why didn't you use `targetIdReq` as in the other endpoints?

Reply:
`targetIdReq` takes a string, so instead of recognising `/api/cohorts/new` like a new endpoint, it searches for the cohort with `id` = 'new'. `cohortId` is an integer and solves this problem.

Reply:
I am aware that this is not optimal, so this should be improved in the future.